src/Core/src/Platform/Android/PickerManager.cs

```diff
@@ -42,7 +42,8 @@ static void OnKeyPress(object? sender, AView.KeyEventArgs e)
 		{
 			if (!AvailableKeys.Contains(e.KeyCode))
 			{
-				e.Handled = false;
+				//To prevent user from entering text when focus is received
+				e.Handled = true;
```

Comment:
To avoid redundant handling of e.Handled can just move it before the if. Always prevent user to enter text and, keep the condition to avoid invoke the CallOnClick method if the key is Tab or arrow keys.

Reply:
@jsuarezruiz , I have updated the code changes. Please let me know if you have any concerns.